content/post/wrong-data.md

```diff
@@ -0,0 +1,55 @@
+I propose there are three common situations in which data is considered wrong, and only one of them would fit a literal definition of “wrong.” They are (in order from least to most common):
+
+<ul>
+<li> The data consumer is using a faulty source to “check” the data.
+<li> The data is literally wrong. 
+<li> The data consumer misunderstands what the data represents. 
+</ul>
+
+## Begin with questions
+Recall the saying, “measure twice, cut once.” 
+
+Ask these two questions to the person who noticed the data being wrong (answer them yourself if you spotted the issue): What result would you expect to see here? Where would you check this number to know it’s right? A number missing a negative sign requires a solution miles away from a bug showing your sales a tenth of the actual number or off by a unit or two.
+
+Every time I look my old code I have new ideas for pretting it up, but when you’re on the hunt for the cause of wrong data keep an eye on the prize. Make time to deal with technical debt another time. 
+
+## Other data is wrong
+This is the least common of the three types of wrong, but common enough that it’s worth checking. One place you might see this is in web site statistics. Suppose you have an internal tracking system tracking site visits, and also use an external tracking tool. On your way to the office an email hits you -- “the site session count is wrong.” (_note: emails arriving between 5am and when you walk into the office are almost never good. No one gets an email saying “Hey we decided to give you a raise.”_). 
+
+Someone shows you how the internal traffic number is 10x more than the external tracking tool. But can you trust that source? Maybe the service was down for an hour last night which caused it’s number to undercount your traffic. A quick check of the recent history might show you if there’s cause for alarm. The real problem might be the tracking of the secondary source.
+
+Maybe someone expected the numbers to match up exactly, which makes sense but also an unrealistic goal. If the two numbers are off by less than a percent it’s likely noise in how the two systems are tracking events. 
+
+I remember someone noting a report was wrong because the sales numbers didn’t make sense. How could we have sales for an item that hadn’t been released yet? Proof the report was wrong? Nope. The server logs showed an affiliate partner had published a direct link to an item we hadn’t made live yet. The report was correct, but by exploring the source of truth we were able to figure out something more meaningful for the company.
+
+## It really is wrong
+If you are giving your screen the “what the heck” stare, and find out that you goofed in your code, give yourself a pat on the back -- seriously. It’s easy enough to write the code and ship it, but to sanity check it and have the business context to see numbers don’t pass the sniff test takes practice. Catching “wrong” because you know what the numbers should be is an accomplishment. 
```

Comment:
I think it's also important to have a plan for communicating with stakeholders who may be affected. Being upfront and honest about mistakes is important for building a healthy analytics culture!